e2e/Makefile

```diff
@@ -10,7 +10,7 @@ TIMEOUT?=168h
 NAMESPACE?=
 CONTEXT?=
 FDB_VERSION?=7.1.31
-PREVIOUS_FDB_VERSION?=6.3.25
+PREVIOUS_FDB_VERSION?=6.3.24
```

Comment:
Why was that changed?

Reply:
IIRC, it was 6.3.25. When I rebased on main branch a week (?) ago, I noticed it was changed.

Reply:
Our CI uses 6.3.25, so I would be more than surprised if that version is missing. I just verified that 6.3.25 is present in our registry we use for testing. Can you please revert this change?